GDJS/Runtime/pixi-renderers/pixi-image-manager.ts

```diff
@@ -309,75 +308,34 @@ namespace gdjs {
      * used by calling `getPIXITexture`.
      * @param onProgress Callback called each time a new file is loaded.
      */
-    loadTextures(
+    async loadTextures(
       onProgress: (loadingCount: integer, totalCount: integer) => void
     ): Promise<integer> {
-      // Construct the list of files to be loaded.
-      // For one loaded file, it can have one or more resources
-      // that use it.
-      const resourceFiles: Record<string, ResourceData[]> = {};
-      for (const res of this._resources.values()) {
-        if (res.file) {
-          if (this._loadedTextures.containsKey(res.name)) {
-            // This resource is already loaded.
-            continue;
+      let loadedCount = 0;
+      await Promise.all(
+        [...this._resources.values()].map(async (resource) => {
+          try {
+            PIXI.Assets.setPreferences({
+              preferWorkers: false,
+              preferCreateImageBitmap: false,
```

Comment:
Why setting these to false?

Reply:
The documentation implies that `crossOrigin` is ignored when `preferCreateImageBitmap` is true.
https://pixijs.download/dev/docs/PIXI.LoadTextureConfig.html

Reply:
Would be good to check with the Pixi team/discord or even try by ourself if loading images from our cloud works with preferCreateImageBitmap. This can be done later.